src/app.rs

```diff
@@ -406,6 +410,20 @@ pub fn app() -> App<'static, 'static> {
              .long("all-pairs")
              .help("Emit a table where each codepoint includes all possible \
                     Simple mappings."));
+    let cmd_case_mapping = SubCommand::with_name("case-mapping")
+        .author(crate_authors!())
+        .version(crate_version!())
+        .template(TEMPLATE_SUB)
+        .about("Create unconditional case mapping tables.")
```

Comment:
Perhaps this could be expanded just a little bit to say something like:

> Create unconditional case mapping tables for upper, lower and title case.

Reply:
Sure. Given the confusion that you had (and that others doubtlessly will have) I think I'll also expand the about blurb to mention that it's used to convert between cases too.